clouds/aws/roles/roles.py

```diff
@@ -19,6 +19,36 @@ def iam_client(region=DEFAULT_AWS_REGION):
     return boto3.client(service_name="iam", region_name=region)
 
 
+def get_roles(client=None):
+    """
+    Get all roles from IAM.
+    Args:
+        client: A boto3 client for IAM.
+
+    Returns:
+         List of roles.
+    """
+    LOGGER.info("Retrieving all roles from IAM.")
+
+    client = client or iam_client()
+
+    response = client.list_roles(MaxItems=1000)
```

Comment:
`1000` - please save and re-use (so if a change is needed in the future, it will be done in one place)

Reply:
@rnetser I saved it as a constant. Is that what you are asking for?